channeld/channeld_wiregen.c

```diff
@@ -1185,4 +1185,4 @@ bool fromwire_send_onionmsg(const tal_t *ctx, const void *p, u8 onion[1366], str
 	return cursor != NULL;
 }
 
-// SHA256STAMP:exp-0-3a681f3ef41ab044c618653409c793a163a7c777181af6c798aff7bc9bef1a77
+// SHA256STAMP:exp-1-78175fb2d475d1cb0c803f952bb07eaa086a0ae0165e2ca2b07ea149a89122a1
```

Comment:
> We can't .gitignore these files, since they're in the repository

That's why I removed them and added them to `.gitignore`.
Is there a specific reason we have/track these generated files in the repository?
From what I can see they are rebuild freshly in under a second if required by the wiregen make targets.